### ISSUE TITLE
Make the app texts more 'global'

### ISSUE THREAD

**Issue by kazlauskis** [BiologicalRecordsCentre/ABLE#743]
From Jacqui
> Because the app is now being used in other countries of the world, would it be possible to change the wording to more global terminology, i.e. Butterfly Monitoring database instead of "European BM"?

We can modify the "European" texts and images for all users or selectively only for users from other continents.

![IMG_1673](https://github.com/user-attachments/assets/37f790e9-f903-4fa4-a3cc-3d6c6a3cca75)
![IMG_1664](https://github.com/user-attachments/assets/16037c84-43df-49b7-b678-4b2ae8e6211b)
![IMG_1674](https://github.com/user-attachments/assets/a46a1f3e-d3c9-43be-ab9b-9f7102cc0702)

**Comment by CrisSevilleja** [comment]
I agree we should include more global context. I was just wondering if it is possible to know whether a user from Europe is using the app. To keep that introduction of the indicator and so on. 
Also, the name eBMS comes from European BMS, so we considered naming it Earth BMS to keep the "e". @DavidRoy

**Comment by braenz** [comment]
It's about three weeks before our Big Backyard Butterfly Count and someone noted today that it was a "European" app. I'm wondering if it's possible to make those small changes soon?

**Comment by DavidRoy** [comment]
Closing as replacing with specific issues such as #755
